src/textUtils.h

```diff
@@ -5,10 +5,29 @@
 
 size_t str_formatAdaAmount(uint64_t amount, char* out, size_t outSize);
 
+#ifdef DEVEL
+void str_traceAdaAmount(const char *prefix, uint64_t amount);
+#define TRACE_ADA_AMOUNT(PREFIX, AMOUNT) \
```

Comment:
just curious, what's the benefit of this debugging macro vs just dumping the prefix and ada amount separately with existing macros? And if it is just saving a line or two of other debugging macros, is it worth it?

Reply:
The problem is a very complicated formatting of ada amounts, a function must be called for that. And you have to allocate and pass a buffer to the function, and combining it with the prefix is annoying in C.